### PR TITLE
App layout

### PR DESCRIPTION
**Pull request by lawther**
This took a bit of mucking about - so much magic in the app-layout stuff :) But now there's a hidey bar with a drawer. The login stuff is hidden in the drawer - probably don't want it there long term, but it at least hides it away while the public conference summary is fetched.

### CODE REVIEW

File: client/conman-app.html [hunk 1]
```diff
@@ -1,37 +1,103 @@
+<script src="bower_components/webcomponentsjs/webcomponents-lite.js"></script>
```
Comment (dstockwell) [inline client/conman-app.html]: Shouldn't need this here. Just once in index.html

File: client/conman-app.html [hunk 2]
```diff
@@ -1,37 +1,103 @@
...
+
 <link rel="import" href="bower_components/polymer/polymer.html">
 
+<link rel="import" href="bower_components/paper-styles/color.html">
+<link rel="import" href="bower_components/font-roboto/roboto.html">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto+Slab:400,700">
```
Comment (dstockwell) [inline client/conman-app.html]: Should this move to index.html where we use roboto?

File: client/conman-app.html [hunk 3]
```diff
@@ -1,37 +1,103 @@
...
+<link rel="import" href="bower_components/app-layout/app-header/app-header.html">
+<link rel="import" href="bower_components/app-layout/app-scroll-effects/app-scroll-effects.html">
 <link rel="import" href="bower_components/app-route/app-route.html">
 <link rel="import" href="bower_components/app-route/app-location.html">
 
 <dom-module id="conman-app">
   <template>
-    <style include="note-app-shared-styles"></style>
+    <style is="custom-style">
+        body {
```
Comment (dstockwell) [inline client/conman-app.html]: Looks like this is also in index.html

File: client/index.html [hunk 4]
```diff
@@ -35,4 +38,10 @@
     database-url="https://con-man.firebaseio.com">
 </firebase-app>
 
-<conman-app></conman-app>
+    <app-location route="{{route}}"></app-location>
```
Comment (dstockwell) [inline client/index.html]: Do we need these here?

File: client/conman-app.html (hunk 1 above)
Comment (lawther) [inline client/conman-app.html]: sg. Deleted.

File: client/conman-app.html (hunk 2 above)
Comment (lawther) [inline client/conman-app.html]: Yep - fair enough. Moved.

File: client/conman-app.html (hunk 3 above)
Comment (lawther) [inline client/conman-app.html]: Good catch. Deleted.

File: client/index.html (hunk 4 above)
Comment (lawther) [inline client/index.html]: Oops, looks like I got too keen with the cuttin' and pastin'. Deleted.

**Pull request merged**